packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -483,6 +489,19 @@ export function createTextNode(
   );
 }
 
+export function resetProperties(
+  domElement: Element,
+  tag: string,
+  rawProps: Object,
+  rootContainerElement: Element | Document | DocumentFragment,
+): void {
+  const attributes = domElement.attributes;
+  while (attributes.length) {
+    domElement.removeAttribute(attributes[0].name);
+  }
+  setInitialProperties(domElement, tag, rawProps, rootContainerElement);
```

Comment:
This function is no longer being inlined on the main path. Seems unfortunate to duplicate it but maybe necessary